CMakeLists.txt

```diff
@@ -46,14 +47,19 @@ else()
     set(CMAKE_INSTALL_PREFIX "/opt/rocm" CACHE PATH "")
 endif()
 
+set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
```

Comment:
We shouldn't set `CMAKE_INSTALL_RPATH`.

Reply:
RPATH is so that it looks for target libraries first inside `/opt/rocm/lib` itself. Otherwise we need to rely on other mechanisms. 
https://linux.die.net/man/3/dlopen

Reply:
First the target libraries should be found since libmigraphx_c.so was also found. Secondly, this sets RPATH to an absolute path, which we shouldn't be doing. Also, `make package` will add the path to ldconf so it will be found. The release packages will set the RPATH/RUNPATH(using relative paths) when building our packages.

If we want to always first search in the same directory as libmigraphx.so, then we can get the path for the .so(using dladdr) and then first check if the target lib exists in that directory. But I think we can do that in a later PR though.

Reply:
Okay. Makes sense. removed `RPATH`.